src/TextInput/TextInput.jsx

```diff
@@ -11,7 +11,7 @@ import "../less/forms.less";
 
 export class TextInput extends React.Component {
   static defaultProps = {
-    placeholderCaps: true,
+    placeholderCaps: false,
```

Comment:
On the docs page for `TextInput`, we can update the default value to be `false`.

Reply:
Good catch!